client/src/components/FavoriteCard/CardInfo/CardInfo.tsx

```diff
@@ -0,0 +1,51 @@
+import { styled } from '@mui/material/styles';
+import { Box, CardContent, Typography } from '@mui/material';
+import React, { FC } from 'react';
+import { Movie } from '../../typedefs/typedefs';
+
+const CardInfoStyle = styled(CardContent)(({ theme }) => ({
+  '&:last-child': {
+    paddingBottom: theme.spacing(2),
+  },
+  display: 'flex',
+  justifyContent: 'space-between',
+  flexDirection: 'column',
+  height: '35%',
+}));
+
+interface Props {
+  movie: Movie,
+}
+
+export const CardInfo: FC<Props> = (props) => {
+  const { movie } = props;
+
+  return (
+    <CardInfoStyle>
+      <Box>
+        <Typography variant="h6" gutterBottom sx={{ fontSize: 14 }}>
+          {movie.title}
+        </Typography>
+
+        <Typography variant="h6" gutterBottom sx={{ fontSize: 10 }}>
+          {typeof movie.genres === 'string' && movie.genres}
```

Comment:
move it to the separate variable
```suggestion
          {isGenresString && movie.genres}
```